pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -115,6 +127,19 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	for index, obj := range supportedTypes {
+		// Validate that GroupVersion is supported on runtime platform
+		ok, err := kubernetes.IsGroupVersionSupported(obj.GroupVersion.String(), obj.Kind())
+		if err != nil {
+			log.Error(err, fmt.Sprintf("Failed to check if GVK '%s' is supported", obj.GroupVersionKind()))
+			continue
+		}
+		supportedTypes[index].GroupVersionSupported = ok
+		if !ok {
```

Comment:
Isn't this `if` & `continue` redundant as nothing proceeds it in the loop?

Reply:
Yeah, it was copy/paste from resource controller where logic a bit complex. Will fix